olive/common/utils.py

```diff
@@ -417,3 +417,13 @@ def find_first_matched_value(original_dict: Dict, keys: Union[str, Tuple, List[s
     if raise_key_error:
         raise KeyError(f"Keys {keys} not found in {original_dict}")
     return None
+
+
+def get_path_by_os(path: Union[Path, str], target_system: OS):
```

Comment:
doesn't `str(path)` also do the same thing? 
![image](https://github.com/microsoft/Olive/assets/94929125/e8147574-65df-4b5b-8da1-69835f74a52c)

Reply:
This was used to get path from a different OS. Forgot to remove this since we go for aml system approach. Will remove this.